.travis.yml

```diff
@@ -183,9 +183,9 @@ before_script:
       && scripts/create_source_tarball.sh)
 
 script:
-    - test $SOLC_EMSCRIPTEN != On || (scripts/test_emscripten.sh)
+    - test $SOLC_EMSCRIPTEN != On || (scripts/test_emscripten.sh) || SOLC_STOREBYTECODE=Off
```

Comment:
That's a very good idea, but won't it make the travis build succeed if the tests fail?

Reply:
Thank you! Great point. fixed